11ty/definitions/fr_FR/femmes-et-personnes-de-couleur.md

```diff
@@ -0,0 +1,30 @@
+---
+title: femmes et personnes de couleur
+slug: femmes-et-personnes-de-couleur
+flag:
+  level: avoid
+defined: true
+excerpt: souvent utilisé comme une expression pour englober "qui n'est pas de couleur blanche, qui n'est pas un homme", cherchant à assurer la solidarité de ces deux groupes ; efface les femmes de couleur, les personnes non binaires
+speech: nom
+alt_words:
+  - personnes de couleur et femmes blanches
+  - personnes de couleur
+  - personnes blanches non-binaires, et femmes blanches
+  - trouver des moyens de recadrer les raisons de l'existence de cette dynamique
+  - ou omettre
+---
+
+souvent utilisé comme une expression pour englober "qui n'est pas de couleur blanche, qui n'est pas un homme", cherchant à assurer la solidarité de ces deux groupes
+
+## Problèmes
+
+Qu'en est-il des femmes de couleur ? En tant que femme de couleur, je suis partagée à la fois entre les femmes et les personnes de couleur.
+
+## Impact
+
+De ce fait, cett expression suscite un sentiment d'effacement chez les femmes de couleur. Elle néglige également les
```

Comment:
Small typo: `cette`

Reply:
All good now, thank you for your sharp eyes!